core/src/test/scala/caliban/TestUtils.scala

```diff
@@ -393,12 +393,13 @@ object TestUtils {
       )
 
       @GQLInterface
-      sealed trait WithNullable {
-        val field: Task[String]
-      }
-      case class WithNonNullable(field: UIO[String]) extends WithNullable
-      case class TestNonNullableObject(nonNullable: WithNonNullable)
-      val resolverNonNullableSubtype = RootResolver(TestNonNullableObject(WithNonNullable(UIO.succeed("a"))))
+      sealed trait WithNullable {}
```

Comment:
Here we want to ensure `field: String` in order for the test assertions to function, but it generates `field: String!`. Not sure how to get around that.